XMonad/Util/Loggers.hs

```diff
@@ -182,6 +186,72 @@ logLayout = withWindowSet $ return . Just . ld
 logCurrent :: Logger
 logCurrent = withWindowSet $ return . Just . W.currentTag
 
+-- | Log the given string, as is.
+logConst :: String -> Logger
+logConst = return . Just
+
+-- | Like they 'maybe' combinator, but for  loggers. If the given logger
```

Comment:
I suppose this is more like `fromMaybe` instead of `maybe` (at least signature wise)?  Also there's a double space between "for" and "loggers"